### DISCUSSION TITLE
Scroll lock

### DISCUSSION THREAD

**Discussion by MyNameTony1000** [KingSora/OverlayScrollbars#500] [Q&A] [answered]
Tell me - how best to block scrolling so that the page freezes in place on any device, when opening modal windows, for example, is there a separate function for this?) I would be very grateful

**Comment by KingSora** [comment, marked as answer]
Good day @MyNameTony1000 

You can simply hide the overflow with `osInstance.options({ overflow: { x: 'hidden', y: 'hidden' })` if the modal is open, and set it back to `scroll` as soon as it is closed again.

**Reply by MyNameTony1000** [reply]
Thank you very much for the help)))
